### PR TITLE
Remove TestCaseView from SmokeTestClassView

### PR DESCRIPTION
**Pull request by eoogbe**
Updates #990

### CODE REVIEW

File: src/main/resources/com/google/api/codegen/php/smoke_test.snip [hunk 1]
```diff
@@ -13,7 +13,7 @@
         /**
          * @@test
          */
-        public function {@smokeTest.method.name}()
+        public function {@smokeTest.apiMethod.name}Test()
```
Comment (garrettjonesgoogle) [inline src/main/resources/com/google/api/codegen/php/smoke_test.snip]: Don't concatenate strings in the templates

**Comment by pongad** [comment]
LGTM for Go

**Comment by eoogbe** [comment]
@michaelbausor Are the output field imports necessary for PHP smoke tests?

**Comment by michaelbausor** [comment]
@eoogbe No, the removed imports in the PHP smoke test LGTM

**Pull request merged**